app/Makefile

```
@@ -106,6 +106,10 @@ stop:
 
 check: format-check lint test
 
+launch-api-only:
+	# Only for local development
+	poetry run launch-api-only -p 8001
```

```suggestion
	$(PY_RUN_CMD) launch-api-only -p 8001
```

This automatically switches between local and in docker depending on the dev's PY_RUN_APPROACH setting

---

I changed it from `$(PY_RUN_CMD)` because I'd have create a docker config to make port 8001 available outside the Docker environment. As the comment says this is `Only for local development`.